include/Mac.h

```diff
@@ -40,6 +40,12 @@ class Mac : public GenericHashEntry {
     char * dhcp; /* Extracted from DHCP dissection */
   } names;
 
+  struct{
+    u_int32_t as_server;
+    u_int32_t as_client;
+  }talkers; /*local talkers*/
+  DeviceTypeCounters dev_t_counters = {0,0,0,0,0,0,0,0,0,0,0,0,0};/*type of the devices this mac talked to*/
+
```

Comment:
Initialization inside class declaration. Awkward and also not accepted by all compilers. Use header files only for declarations. Do the necessary initializations inside class members definitions in the corresponding cpp file.